### PR TITLE
add Primary component

### PR DESCRIPTION
**Pull request by CedricNtwari**
Short explanation of the things the PR is about...

### Review Instructions

Add special instructions a reviewer has to know about here (e.g. "to reproduce the issue click this and fill out that and submit the other...").

### Checklist

- [ ] Unit/feature tests are implemented
- [ ] Tested on a real mobile device
- [ ] Performance check of relevant pages done: [https://www.webpagetest.org](https://www.webpagetest.org)

### CODE REVIEW

File: components/Footer.vue [hunk 1]
```diff
@@ -20,16 +20,17 @@ export default {
...
 .footer {
   position: fixed;
   padding: 10px 10px 0px 10px;
   bottom: 0;
   width: 100%;
   height: 40px;
   background: grey;
+
+  &__copyright {
+  font-size: rem(14);
```
Comment (CedricNtwari) [inline components/Footer.vue]: @mishschmid Testing global styles

File: components/NavBar.vue [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
 </template>
 
-<style scoped>
+<style lang="scss">
 .brand {
   font-family: "Montserrat", sans-serif;
   font-weight: 700;
   font-size: 1.5em;
-  color: #39b982;
+  color: $color-green;
```
Comment (CedricNtwari) [inline components/NavBar.vue:16]: @mishschmid Testing global styles

File: components/NavBar.vue [hunk 3]
```diff
@@ -24,15 +24,20 @@
...
+  border-bottom: solid rem(2) $color-green;
 }
 .nav a {
   display: inline-block;
+  color: $color-green;
+
+  @media ($mobile-up) {
+    display: flex;
+  }
```
Comment (CedricNtwari) [inline components/NavBar.vue:41]: @mishschmid same here

File: nuxt.config.ts [hunk 4]
```diff
@@ -1,6 +1,22 @@
...
+  vite: {
+    css: {
+        preprocessorOptions: {
+            scss: {
+                additionalData: '@import "@/styles/index.scss";',
+            },
+        },
+    },
+}
 })
```
Comment (CedricNtwari) [inline nuxt.config.ts:22]: @mishschmid So one thing interesting though is how to configure `@nuxtjs/style-resources`. Found this solution here https://githubmemory.com/repo/nuxt-community/style-resources-module/issues/190 

Plus I had to downgrade `sass:1.32.13` to fix these kind of warnings `DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.`

**Comment by CedricNtwari** [comment]
@mishschmid Sorry PR is a mess. Just want you to show how @nuxtjs/style-resources is set up.

File: README.md [hunk 5]
```diff
@@ -27,4 +39,5 @@ yarn build
 ```
 
 Checkout the [deployment documentation](https://v3.nuxtjs.org/docs/deployment).# nuxt3-app
-# nuxt3-app
+
+# Bamboo-nuxt3-app
```
Comment (CedricNtwari) [inline README.md:43]: @mishschmid Just beautify the readme. Is this clear for someone who wants to clone this project? anything missing? You never know who is reading this!😀

Comment (mishschmid) [reply README.md:43]: Looking good 💄 !

I personally would just remove the "nuxt 3 app" part of it. No need to mention that in the title. You are also not saying welcome to the "Bamboo node prettier graphql vue app" 😄  

So I would just go with Bamboo or Bamboo Bikes or whatever the official name is. But totally up to you, just a personal opinion.

Comment (CedricNtwari) [reply README.md:43]: Good point!😀

**Pull request merged**